demo/package.json

```diff
@@ -12,27 +12,27 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^8.2.13",
-    "@angular/compiler": "^8.2.13",
-    "@angular/core": "^8.2.13",
-    "@angular/forms": "^8.2.13",
-    "@angular/platform-browser": "^8.2.13",
-    "@angular/platform-browser-dynamic": "^8.2.13",
-    "@angular/router": "^8.2.13",
+    "@angular/common": "^9.1.13",
+    "@angular/compiler": "^9.1.13",
+    "@angular/core": "^9.1.13",
+    "@angular/forms": "^9.1.13",
+    "@angular/platform-browser": "^9.1.13",
+    "@angular/platform-browser-dynamic": "^9.1.13",
+    "@angular/router": "^9.1.13",
     "core-js": "^3.3.6",
     "ngx-datetime-picker": "^2.3.0",
     "rxjs": "^6.5.3",
     "rxjs-compat": "^6.5.3",
     "tslib": "^1.10.0",
-    "zone.js": "~0.9.1"
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.803.17",
-    "@angular/cli": "^8.3.17",
-    "@angular/compiler-cli": "^8.2.13",
+    "@angular-devkit/build-angular": "~0.901.15",
+    "@angular/cli": "^9.1.15",
+    "@angular/compiler-cli": "^9.1.13",
     "@types/jasmine": "3.4.5",
-    "@types/node": "^12.12.5",
-    "codelyzer": "^5.2.0",
+    "@types/node": "^12.11.1",
+    "codelyzer": "^5.1.2",
```

Comment:
Same here too - this one is less of a concern though. This is a tslint dependency that'll go away soon.